uiowa/uiowa
# Pharmacy map of Palliative Care Graduates

Relates to #3624


## Story

As a site editor, I can maintain a list of Palliative Care Graduates and the territories they relate to and display the results on a map within layout builder.

## How to test

- `blt frontend`
- `drush @pharmacy.local config:import --source ../config/pharmacy.uiowa.edu --partial`
- `drush @pharmacy.local en pharmacy_core`
- Create at least one person with the Palliative Care Graduate person type designation and a territory.
- Find the map and the list within the more section of layout builder
- See that the person and their related territory are display appropriately within the list and map.


## ToDo
For reference: https://github.com/uiowa/uiowa/pull/2521/files

- [x] Get person type filter on view to work (the string is not working) or switch to tag like stated in #3624
- [x] yarn workspace
- [x] libraries
- [x] map js
- [x] block calling map js and getting variables (allowed by lb page default) @bspeare
